butterknife-gradle-plugin/src/main/java/butterknife/plugin/FinalRClassBuilder.java

```diff
@@ -26,7 +26,8 @@
 public final class FinalRClassBuilder {
   private static final String SUPPORT_ANNOTATION_PACKAGE = "android.support.annotation";
   private static final String[] SUPPORTED_TYPES = {
```

Comment:
Do you want to just get rid of `SUPPORTED_TYPES` and allow all types? This would make it more future proof , and it's what gautam put up on the butterknife repo